Source/FullScreenPokemon.ts

```diff
@@ -1351,10 +1351,14 @@ module FullScreenPokemon {
          * Starts the Player cycling if the current Area allows it.
          *
          * @param thing   A Player to start cycling.
-         * @param area   The current Area.
+         * @param message   Whether to display a message box.
          * @returns Whether the properties were changed.
          */
-        startCycling(thing: IPlayer): boolean {
+        startCycling(thing: IPlayer, message?: boolean): boolean {
+            if (thing.cycling) {
```

Comment:
Merge this with the next if statement.

This function returns `boolean` so don't forget to respect that.